classes/class-bsf-custom-fonts-render.php

```diff
@@ -277,22 +278,24 @@ public function enque_data() {
 		 * @since 1.0.4
 		 */
 		public function add_style() {
-			$font_styles = '';
-			$query_posts = $this->get_existing_font_posts();
 
-			if ( $query_posts ) {
-				foreach ( $query_posts as $key => $post_id ) {
-					$font_styles .= get_post_meta( $post_id, 'fonts-face', true );
+			if ( class_exists( 'Thrive_Product_Manager' ) || class_exists( 'TVA_Const' ) ) {
```

Comment:
Make a static callback function which will return `class_exists( 'Thrive_Product_Manager' ) || class_exists( 'TVA_Const' )` and use that callback at these 2 locations.

Reply:
Added static callback function as per suggestion